--- a/dlls/win32u/vulkan.c
+++ b/dlls/win32u/vulkan.c
@@ -3002,12 +3002,17 @@ static void vulkan_init_once(void)
     VkResult res;
     const char *libvulkan, *graphics_backend;
 
+#ifdef SONAME_LIBVULKAN
     /* CW HACK 25909: Allow specifying libvulkan */
     if (!(libvulkan = getenv( "CX_LIBVULKAN" )) ||
         !(graphics_backend = getenv( "CX_ACTIVE_GRAPHICS_BACKEND" )) ||
         strcmp( graphics_backend, "wined3d" ))
         libvulkan = SONAME_LIBVULKAN;
     else
         ERR( "Using %s\n", libvulkan );
+#else
+    ERR( "Vulkan support is unavailable in this build.\n" );
+    return;
+#endif
 
     if (!(vulkan_handle = dlopen( libvulkan, RTLD_NOW )))
